pages/linux/brew.md

```diff
@@ -14,14 +14,18 @@
 
 `brew list`
 
-- Update an installed formula (passing no formula updates all installed formulae):
+- Update an installed formula (passing no `{{formula}}` updates all installed formulae):
 
 `brew upgrade {{formula}}`
 
-- Update brew itself:
+- Fetch the newest version of Linuxbrew and all formulae from GitHub:
```

Comment:
The Linux version of Homebrew is called "Linuxbrew". You can see this term is already being used in `pages/linux/brew.md`.